pkg/services/ngalert/notifier/external_alertmanager.go

```diff
@@ -126,7 +150,7 @@ func (am *externalAlertmanager) StopAndWait() {
 }
 
 func (am *externalAlertmanager) Ready() bool {
-	return false
+	return true
```

Comment:
This change doesn't really matter as this struct is not currently being used. Once I implement this method this will change again.